module/VuFind/src/VuFind/AjaxHandler/GetResolverLinks.php

```diff
@@ -136,6 +136,7 @@ public function handleRequest(Params $params)
                     // Special case -- modify DOI text for special display:
                     $link['title'] = $this->translate('Get full text');
                     $link['coverage'] = '';
+                    // no break
```

Comment:
I'd prefer repeating the code from the `default` case and adding a break here, but either way works for me.